tests/integration/components/sl-translate-test.js

```diff
@@ -0,0 +1,128 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-translate', 'Integration | Component | sl translate', {
+    beforeEach() {
+        this.container.lookup( 'service:sl-translate' )
+            .setDictionary(
+                Ember.Object.create({
+                    'SIMPLE_KEY': 'I have been translated',
+                    'SINGULAR_KEY': 'View my family',
+                    'PLURAL_KEY': 'View my families',
+                    'REPLACED_KEY': 'I have replaced {0} and {1}'
+                })
+            );
+    },
+
+    integration: true
+});
+
+test( 'Simple Key Translation', function( assert ) {
+    assert.expect( 1 );
```

Comment:
In the article http://alisdair.mcdiarmid.org/ember-component-integration-tests/ and corresponding tests: https://github.com/alisdair/ember-component-integration-tests/blob/master/tests/integration/components/alert-banner-test.js the assert.expect() is used. I followed this pattern as well, however, when I removed the assert.expect() from my tests they still worked fine.